js/two-factor.js

```diff
@@ -0,0 +1,35 @@
+/* global ajaxurl */
```

Comment:
JSCS should be throwing errors in Travis for a lot of the spacing between parentheses in this file. Not sure why it didn't.